README.md

```diff
@@ -37,411 +37,41 @@ Special acknowledgment to the authors of [Playwright](https://playwright.dev/) a
-with the Locator API, even if the underlying frame
-navigates, locators will continue to work.
-
-The Locator API can also help you abstract a page to simplify testing.
-To do that, you can use a pattern called the Page Object Model.
-You can see an example [here](examples/locator_pom.js).
-
-```js
-import { chromium } from 'k6/x/browser';
-
-export default function () {
-  const browser = chromium.launch({
-    headless: false,
-  });
-  const context = browser.newContext();
-  const page = context.newPage();
-
-  page.goto("https://test.k6.io/flip_coin.php", {
-    waitUntil: "networkidle",
-  });
-
-  /*
-  In this example, we will use two locators, matching a
-  different betting button on the page. If you were to query
-  the buttons once and save them as below, you would see an
-  error after the initial navigation. Try it!
-
-    const heads = page.$("input[value='Bet on heads!']");
-    const tails = page.$("input[value='Bet on tails!']");
-
-  The Locator API allows you to get a fresh element handle each
-  time you use one of the locator methods. And, you can carry a
-  locator across frame navigations. Let's create two locators;
-  each locates a button on the page.
-  */
-  const heads = page.locator("input[value='Bet on heads!']");
-  const tails = page.locator("input[value='Bet on tails!']");
-
-  const currentBet = page.locator("//p[starts-with(text(),'Your bet: ')]");
-
-  // In the following Promise.all the tails locator clicks
-  // on the tails button by using the locator's selector.
-  // Since clicking on each button causes page navigation,
-  // waitForNavigation is needed -- this is because the page
-  // won't be ready until the navigation completes.
-  // Setting up the waitForNavigation first before the click
-  // is important to avoid race conditions.
-  Promise.all([
-    page.waitForNavigation(),
-    tails.click(),
-  ]).then(() => {
-    console.log(currentBet.innerText());
-    // the heads locator clicks on the heads button
-    // by using the locator's selector.
-    return Promise.all([
-      page.waitForNavigation(),
-      heads.click(),
-    ]);
-  }).then(() => {
-    console.log(currentBet.innerText());
-    return Promise.all([
-      page.waitForNavigation(),
-      tails.click(),
-    ]);
-  }).finally(() => {
-    console.log(currentBet.innerText());
-    page.close();
-    browser.close();
-  })
-}
-```
-
-### Running examples in a Docker container
-
-The examples above are also available as standalone files in the [`examples` directory](/examples). You can run them in a Docker container using Docker Compose with:
+- [Releases](https://github.com/grafana/xk6-browser/releases)
+- [API Documentation](https://k6.io/docs/javascript-api/xk6-browser/api/) - This documents how to use our API and its current state with xk6-browser.
+- [Examples](/examples) - The examples from our API documentation are also available as standalone files. You can run them in a Docker container using Docker Compose with:
```

Comment:
Think we should either move the docker stuff to the docs in docs.k6.io, or create a new subheading somewhere in the readme.

Reply:
Will move it to the docs 👍 Also, see this [comment](https://github.com/grafana/xk6-browser/pull/656#discussion_r1026544499) .

Reply:
Created https://github.com/grafana/k6-docs/pull/907